src/cluster.c

```diff
@@ -108,6 +108,22 @@ ConnectionType *connTypeOfCluster(void) {
     return connectionTypeTcp();
 }
 
+/* delete unowned keys from database at server start up after loading persistence files. */
```

Comment:
nit: We can write the first letter in uppercase.

```suggestion 
/* Delete unowned keys from database at server start up after loading persistence files. */
```